src/frontend/jest.config.js

```diff
@@ -4,5 +4,7 @@ module.exports = {
   setupFilesAfterEnv: ['./testSetup.ts', 'regenerator-runtime/runtime'],
   testMatch: [__dirname + '/js/**/*.spec.+(ts|tsx|js)'],
   testURL: 'https://localhost',
-  transformIgnorePatterns: ['node_modules/(?!(lodash-es)/)'],
+  transformIgnorePatterns: [
+    '/node_modules/(?!(' + 'lodash-es|' + 'draft-js-latex-plugin' + ')/)',
```

Comment:
https://github.com/openfun/richie/blob/9bed95e7320d1d8c2e7f123c7b638f4a6b612f91/src/frontend/jest.config.js#L17

This syntax should work.
```suggestion
    '/node_modules/(?!(lodash-es|draft-js-latex-plugin)/)',
```